js/interface.js

```diff
@@ -1,8 +1,6 @@
 Fliplet().then(function () {
   var appId = Fliplet.Env.get('appId');
 
-  $(window).on('resize', Fliplet.Widget.autosize);
```

Comment:
This is still recommended to keep. I always have a `Fliplet.Widget.autosize()` for `window.resize` event

Reply:
Ok. Thanks.